SalesforceMobileSDK.xcworkspace/xcshareddata/xcschemes/UnitTests.xcscheme

```diff
@@ -4,7 +4,7 @@
    version = "1.3">
    <BuildAction
       parallelizeBuildables = "NO"
-      buildImplicitDependencies = "NO">
+      buildImplicitDependencies = "YES">
```

Comment:
@bhariharan Is there any reason this was unchecked?  Static analysis fails (because it can't link RCTWebSocket) with it set to "NO".

Reply:
No reason. Feel free to flip the switch.